comfy_cli/command/assets_library.py

```
@@ -0,0 +1,111 @@
+
+
+@app.command("ls", help="List your assets on Comfy Cloud.")
+@tracking.track_command("assets")
+def ls_cmd(
+    name: Annotated[
+        str | None,
+        typer.Option("--name", show_default=False, help="Case-insensitive substring match on asset name."),
+    ] = None,
+    tags: Annotated[
+        str | None,
+        typer.Option("--tags", show_default=False, help="Comma-separated tags; assets must have ALL of them (e.g. input,output)."),
```

Pylint (4.0.6)

[convention] Line too long (131/120)

(C0301)